### ISSUE TITLE
 add watermark to a video with the given font.

### ISSUE THREAD

**Issue by tejan-singh** [sanscript-tech/hacking-tools-scripts#257]
use openCV

**Comment by kuldip11** [comment]
Assigned me for python

**Comment by tejan-singh** [comment]
@kuldip11 you can work on this issue.

**Comment by tejan-singh** [comment]
@kuldip11 are you working on this?

**Comment by Ninjavin** [comment]
@tejan-singh I would like to work on this in Python if this is unassigned.

**Comment by kuldip11** [comment]
Yes i am currently working on it and try to squash my old commit... and i will submit it by tomorrow, i mistakenly got unassigned.

**Comment by tejan-singh** [comment]
@Ninjavin issue was assigned for python. Can you work in other language.

**Comment by SANKET7738** [comment]
@Ninjavin are you going to take up this issue? otherwise, I will close the issue.

**Comment by Ninjavin** [comment]
Hey @SANKET7738 Sorry for replying late. I don't think I will take this issue in any other language. You may close it.
